spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTMPeephole.scala

```diff
@@ -89,28 +92,26 @@ class LSTMPeephole[T : ClassTag] (
   }
 
   def buildInputGate(): Sequential[T] = {
-    inputGate = buildGate()
+    inputGate = buildGate(2, 1, hiddenSize)
```

Comment:
I think it would be better if we define a constant similar variable, set it as 2 and use it here